### PR TITLE
added controller mapping

### PR DESCRIPTION
**Pull request by ihor-il**
Hotfix for #19 where controller mapping was deleted while merging

### CODE REVIEW

**Review by DaniilPoiarkov — DISMISSED** [review]

File: backend/Tasque.WebAPI/Program.cs [hunk 1]
```diff
@@ -40,6 +40,7 @@
 app.UseStaticFiles();
 
 app.UseRouting();
+app.MapControllers();
```
Comment (Its-OP) [inline backend/Tasque.WebAPI/Program.cs]: MapControllers is a middleware of type 'Endpoint'. It must be the last one in the middleware list. Use this article for more details:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-6.0#middleware-order

**Review by Its-OP — CHANGES_REQUESTED** [review]

File: backend/Tasque.WebAPI/Program.cs (hunk 1 above)
Comment (ihor-il) [reply backend/Tasque.WebAPI/Program.cs]: right before app.Run()?

Comment (Its-OP) [reply backend/Tasque.WebAPI/Program.cs]: yep

**Review by ismv — APPROVED** [review]
LGTM

**Review by Its-OP — APPROVED** [review]

**Pull request merged**